ktor-client/ktor-client-plugins/ktor-client-encoding/common/src/ContentEncoding.kt

```diff
@@ -64,7 +66,7 @@ public class ContentEncoding private constructor(
 
             LOGGER.trace("Recoding response with $encoder for ${response.call.request.url}")
             with(encoder) {
-                current = decode(current)
+                current = decode(current, response.coroutineContext)
```

Comment:
Please add tests with decompressing on completed response: we can receive `HttpResponse` and next receive String or some other type on it

Reply:
we do this in `ContentEncodingRequestBodyTest`

Reply:
`ContentEncodingTest` has such tests